demos/node-classification/cluster-gcn/cluster-gcn-node-classification.ipynb

```diff
@@ -358,12 +358,12 @@
      "data": {
```

Comment:
<p>Why is the previous cell removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
the previous cell wasn't actually affecting anything downstream and was editing copies of a DataFrame causing the warnings! So I just removed it

Reply:
Hmm I wonder if it's considered bad data science practice to not separate the ground truth labels from the data though?  Is there a way we can still do that without producing warnings?

Reply:
fair point, it should be pretty easy to to do. i'll give it a go

Reply:
done! I added the cell back in but used `train_data = train_data.drop(..., inplace=False, ...)`